Ginger/GingerCoreCommon/Repository/BusinessFlowLib/Activity.cs

```diff
@@ -372,8 +372,36 @@ public ObservableList<IAct> Acts
             }
         }
 
+        private ObservableList<VariableBase> mVariables;
+        /// <summary>
+        /// Been used to identify if Activity Variables were lazy loaded already or not
+        /// </summary>
+        public bool VariablesLazyLoad { get { return (mVariables != null) ? mVariables.LazyLoad : false; } }
```

Comment:
Codacy found an issue: [Remove the unnecessary Boolean literal(s).](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5970337)